src/components/biodiversity-sidebar-card/biodiversity-sidebar-card-component.jsx

```diff
@@ -5,40 +5,46 @@ import Tabs from 'components/tabs';
 import CategoryBox from 'components/category-box';
 import BiodiversityLayers from 'components/biodiversity-layers';
 import { biodiversityCategories } from 'constants/mol-layers-configs';
-import styles from './biodiversity-sidebar-card-styles.module.scss'
+import { LAYER_VARIANTS } from 'constants/landscape-view-constants';
+import capitalize from 'lodash/capitalize';
+import styles from './biodiversity-sidebar-card-styles.module.scss';
 
-const biodiversity = LAYERS_CATEGORIES.BIODIVERSITY;
-
-const BiodiversitySidebarCardComponent = ({activeLayers, countedActiveLayers, map, view}) => {
-  const [isOpen, setOpen] = useState(false)
+const BiodiversitySidebarCardComponent = ({
+  activeLayers,
+  countedActiveLayers,
+  map,
+  view,
+  changeUI,
+  biodiversityLayerVariant
+}) => {
+  const [isOpen, setOpen] = useState(false);
   const handleBoxClick = () => setOpen(!isOpen);
+
   return (
     <div className={styles.sidebarCardContainer}>
       <CategoryBox
         title="mapping"
-        category={biodiversity}
-        counter={countedActiveLayers[biodiversity]}
+        category={LAYERS_CATEGORIES.BIODIVERSITY}
+        counter={countedActiveLayers[LAYERS_CATEGORIES.BIODIVERSITY]}
         handleBoxClick={handleBoxClick}
         isOpen={isOpen}
       />
       <div
         className={cx(styles.layersTogglesContainer, { [styles.open]: isOpen })}
```

Comment:
The `open` styles need to be updated. They have a fixed `height` in order to allow the container to animate that property; but since the `CategoryBox` does not have the same content in all of the variants maybe this value needs to be updated acordingly (if not the box is too big when priority layers are active)